spec/lib/tasks/volume_tracking.rake_spec.rb

```diff
@@ -22,7 +24,7 @@
       well_aliquots = create_list(:aliquot, 10, used_by: create(:pacbio_well), volume: 10, concentration: 10)
 
       # We shouldnt change the amount of aliquots
-      expect { Rake::Task['volume_tracking:clear_well_aliquot_volume'].invoke }.not_to change(Aliquot, :count) and output(
+      expect { Rake::Task['volume_tracking:clear_well_aliquot_volume'].invoke }.to not_change(Aliquot, :count).and output(
```

Comment:
Why is this change necessary if they both achieve the same thing, I hope?

Reply:
Yeah they are the same thing, this is a slightly unrelated change but this test was outputting text when running the test suite which we don't want and this fixes it.